src/ru/nsu/ccfit/zuev/skins/SkinReader.java

```diff
@@ -0,0 +1,22 @@
+package ru.nsu.ccfit.zuev.skins;
+
+import org.json.JSONObject;
+
+public abstract class SkinReader {
```

Comment:
Do you see more potential use cases for this class? Because it looks like this is only used in `SkinJsonReader` and its abstract methods are quite restrictive.

Reply:
Yeah maybe in the future for use of other format such as skin.ini but it will require modifications still i guess